lib/iris/tests/unit/cube/test_Cube__operators.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 import numpy.ma as ma
 import biggus
+from biggus._init import _Elementwise as Elementwise
```

Comment:
i do not like importing a private class definition and changing its name to remove the leading underscore